addons/hr/views/hr_employee_views.xml

```diff
@@ -73,7 +73,7 @@
                             </button>
                         </div>
                         <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
-                        <field name="image_1920" widget='image' class="oe_avatar" options='{"zoom": true, "preview_image":"image_128"}'/>
+                        <field name="image_1920" widget='image' class="oe_avatar" placeholder="/base/static/img/avatar_grey.png" options='{"zoom": true, "preview_image":"image_128"}'/>
```

Comment:
Assuming this is the form view exception where avatar is not used, shouldn't this use as placeholder what was initially in `_get_placeholder_filename` from employee, that you removed?

Reply:
`_get_placeholder_filename` is used when web/image doesn't find a value. However, in this view the image widget sets the placeholder directly, without calling the web/image route